lib/services/irc.rb

```diff
@@ -137,6 +137,31 @@ def irc_puts(command, debug_command=command)
     writable_irc.puts command
   end
 
+  def irc_realname
+    repo_owner = payload["repository"]["owner"]["name"]
+    repo_name = payload["repository"]["name"]
+
+    "GitHub IRCBot - #{repo_owner}" + (repo_public? ? "/#{repo_name}" : "")
+  end
+
+  def repo_public?
```

Comment:
When I wrote https://github.com/Zarthus/github-services/commit/f695311650a2017d4ee519b07aef531cf238f808 the tests seemed to pass, further verified by https://github.com/Zarthus/github-services/blob/master/lib/service/events/status_helpers.rb#L89 and https://github.com/Zarthus/github-services/blob/master/lib/service/events/deployment_helpers.rb#L34 having private fields, I probably just looked over them. 

I'm not entirely sure why the commit is not showing up in this PR. Though. Is there a way to tell github to perform a re-check?